Cargo.toml

```diff
@@ -86,7 +86,7 @@ full = [
 ]
 
 # HTTP versions
-http1 = []
+http1 = ["tokio/time"]
```

Comment:
In order to keep the runtime stuff optional, I think this should be part of the `runtime` feature, similar to the http2 keep-alive code as well.